docs/plugins/web.rst

```diff
@@ -160,6 +162,13 @@ response includes all the items requested. If a track is not found it is silentl
 dropped from the response.
 
 
+``GET /item/by_path/...``
++++++++++++++++++++++
```

Comment:
For symmetry with the endpoint called `query`, maybe this one should be called `path` instead of `by_path`?